### PR TITLE
Making the readme more descriptive

### PR DESCRIPTION
**Pull request by animeshsingh**
around the scope, as well as fixed some syntax and semantic issues

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/39)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
+KFServing provides a [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML Models on arbitrary frameworks. It aims to solve majority of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and custom containers. Using KFServing you should be able to deploy, scale up or down to zero and perform A/B tests on your models in a consistent and standardized way.
 
-A KFService encapsulates the complexity of autoscaling, networking, health checking, server configuration, and more, to provide customers with a simple and seamless experience when deploying models.
+A KFService encapsulates the complexity of autoscaling, networking, health checking, server configuration, and more, to provide customers with a simple and seamless experience when deploying models.
```
Comment (ellis-bigelow) [inline README.md]: You added a space.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
-A KFService encapsulates the complexity of autoscaling, networking, health checking, server configuration, and more, to provide customers with a simple and seamless experience when deploying models.
+A KFService encapsulates the complexity of autoscaling, networking, health checking, server configuration, and more, to provide customers with a simple and seamless experience when deploying models.  
 
 In the future, we hope to support more advanced use cases such as skew detection, explainability, and performance profiling across infrastructure configurations.
 
-This project is an evolution of the original proposal: https://github.com/kubeflow/kubeflow/issues/2306
\ No newline at end of file
+This project is an evolution of the original proposal: https://github.com/kubeflow/kubeflow/issues/2306
```
Comment (ellis-bigelow) [inline README.md]: How did this change?

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # KFServing
-KF Serving provides a Custom Resource Definition for serving ML Models on arbitrary frameworks. It aims to solve 80% of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and Custom containers.
+KFServing provides a [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML Models on arbitrary frameworks. It aims to solve majority of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and custom containers. Using KFServing you should be able to deploy, scale up or down to zero and perform A/B tests on your models in a consistent and standardized way.
```
Comment (ellis-bigelow) [inline README.md]: I think 80% conveys the intended meaning better than "majority". It's a good framework for thinking following the 80-20 rule.

Comment (ellis-bigelow) [inline README.md]: AB testing is not supported by KFServing. We support Canarying.


```suggestion
KFServing provides a [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML Models on arbitrary frameworks. It aims to solve 80% of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and custom containers. KFServing brings cutting edge serving features like Scale to Zero and Canary Rollouts to your ML deployments.
```

Comment (animeshsingh) [reply README.md]: Yeah - its just that it conveys that we have 100% of the use cases, covering 80% here. A number to me looked a bit out of place.

File: README.md (hunk 2 above)
Comment (animeshsingh) [reply README.md]: not sure - harmless though

File: README.md (hunk 3 above)
Comment (ellis-bigelow) [reply README.md]: Should we add a link to https://en.wikipedia.org/wiki/Pareto_principle

Comment (animeshsingh) [reply README.md]: Think it would distract from readability.  More layman language, the better for first page.

Comment (animeshsingh) [reply README.md]: Also if i see a hyperlink there, expectation would be it points me to all the model serving use cases detailed, and points what all we are covering

**Comment by ellis-bigelow** [comment]
@animeshsingh , do you need to sign the CLA?
/lgtm
/approve

**Comment by animeshsingh** [comment]
I have contributed to KF in past. Needs your consent as an 'author' because I accepted your revised changes.

**Comment by animeshsingh** [comment]
@ellis-bigelow you may just need to type 'i consent' and this should take care of it.

**Comment by ellis-bigelow** [comment]
I consent

**Comment by ellis-bigelow** [comment]
Thanks Animesh

**Comment by ellis-bigelow** [comment]
Save us @googlebot !

**Comment by ellis-bigelow** [comment]
Aha. I figured it out:
(<Unknown>)  singhan@us.ibm.com: need_author_consent
(<Unknown>)  ellisbigelow@google.com: ok

**Comment by animeshsingh** [comment]
I have the CLA, and have merged PRs as I mentioned. Now let me see if me saying 'I consent' makes a change, else maybe @jlewi can help here.

**Comment by ellis-bigelow** [comment]
I can (and have been) trigger re-scans here: https://github.corp.google.com/prinfo?q=http://github.com/kubeflow/kfserving/pull/39

**Comment by ellis-bigelow** [comment]
Try "I consent" as the only body of a comment.

**Comment by animeshsingh** [comment]
I consent

**Comment by animeshsingh** [comment]
Maybe you can try /retest. If not, maybe adding me as an approver might do the trick (though shouldn't be needed)

**Comment by animeshsingh** [comment]
going to close and resubmit a clean. not worth the effort here. BTW do we still want to stick with 80%?

**Comment by ellis-bigelow** [comment]
That would be my preference. It's been a useful metaphor for communicating with various parties.

**Pull request closed without merging**